lib/handlers/noscript.ts

```diff
@@ -37,6 +36,9 @@ import {StorageBase} from '../storage/index.js';
 import {CompileHandler} from './compile.js';
 import {cached, csp} from './middleware.js';
 
+const router = express.Router();
```

Comment:
The router for noscript is in the constructor of the class NoScriptHandler, not outside. So this code should go inside the constructor, I think?